### ISSUE TITLE
Dependencies Wrong in 1.0.12 release on yarnpkg

### ISSUE THREAD

**Issue by alexcurtin** [RoadieHQ/roadie-backstage-plugins#147]
The `@backstage/core-components` should be at release `^0.3.1` in the `@roadiehq/backstage-plugin-github-pull-requests@^1.0.12"` package, but when you pull down that package, that dependency is pinned to `0.2.0` which is causing issues with backstage.

see: https://cdn.jsdelivr.net/npm/@roadiehq/backstage-plugin-github-pull-requests@1.0.12/package.json

and my yarn.lock after specifying @roadiehq/backstage-plugin-github-pull-requests@^1.0.12

```
"@roadiehq/backstage-plugin-github-pull-requests@^1.0.12":
  version "1.0.12"
  resolved "https://registry.yarnpkg.com/@roadiehq/backstage-plugin-github-pull-requests/-/backstage-plugin-github-pull-requests-1.0.12.tgz#96a3afbce27bfdc75d766c800a317baf4588405c"
  integrity sha512-LscobaeWIatTFqQnBqF+xG6fsu0rPrU7z+sOrbjGCuIo2a4sYhdgnbwYqVWTH2LRb6o8G4K24CYFYC1uQkNkCw==
  dependencies:
    "@backstage/catalog-model" "^0.9.0"
    "@backstage/core-app-api" "^0.1.3"
    "@backstage/core-components" "^0.2.0"
    "@backstage/core-plugin-api" "^0.1.3"
    "@backstage/plugin-catalog-react" "^0.4.0"
    "@material-ui/core" "^4.11.0"
    "@material-ui/icons" "^4.9.1"
    "@octokit/rest" "^18.5.3"
    "@octokit/types" "^5.0.1"
    "@types/node-fetch" "^2.5.7"
    history "^5.0.0"
    moment "^2.27.0"
    node-fetch "^2.6.1"
    react "^16.12.0"
    react-dom "^16.12.0"
    react-router "6.0.0-beta.0"
    react-use "^17.2.4"
    ```

**Comment by martina-if** [comment]
https://app.clubhouse.io/larder/story/3536

**Comment by alexcurtin** [comment]
This was fixed in release 1.0.13
